addons/im_livechat/static/src/discuss_app/discuss_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="im_livechat.Discuss" t-inherit="mail.Discuss" t-inherit-mode="extension" owl="1">
+        <xpath expr="//*[contains(@class, 'o-mail-Discuss-threadAvatar')]" position="attributes">
+            <attribute name="t-if">thread and thread.model === 'discuss.channel' and thread.type !== 'livechat'</attribute>
```

Comment:
Maybe it would be better if the basic case listed the cases that are allowed, rather than having an override to remove the element.